EFProvider/LocalizationContext.cs

```diff
@@ -8,6 +9,107 @@
 
 namespace Knoema.Localization.EFProvider
 {
+#if false
```

Comment:
what's the point in #if false?

Reply:
Can change to 'true' and able to run locally without DB connection required